targets/xorg

```diff
@@ -181,7 +181,7 @@ if [ -n "$backport" ]; then
         "xserver-xorg-input-synaptics$backport" $installvideodrivers
 fi
 
-if release -lt stretch -lt kali-rolling -lt yakkety; then
+if release -lt stretch -lt kali-rolling -lt yakkety -lt tumbleweed; then
```

Comment:
`-lt tumbleweed` means any opensuse release that is earlier than tumbleweed, not including tumbleweed itself.  Is that what you meant?